packages/extension/src/utils/payment.ts

```diff
@@ -76,11 +76,22 @@ export const buildDestinationTag = (destinationTag: string | undefined): number
   return Number(destinationTag);
 };
 
+export const checkFee = (fee: string | null) => {
```

Comment:
Can you please add this missing unit test?

Reply:
Done